aio/firebase.json

```
@@ -94,18 +94,18 @@
 
       // URLs that use the old scheme of adding the type to the end (e.g. `SomeClass-class`)
       // (Exclude disambiguated URLs that might be suffixed with `-\d+` (e.g. `SomeClass-1`))
-      {"type": 301, "regex": "^/api/(?P<package>[^/]+)/(?P<api>[^/]+)-\\D*$", "destination": "/api/:package/:api"},
+      {"type": 301, "regex": "^/api/(?P<package>[^/]+)/(?P<api>[^/-]+)-(class|directive|var|interface|function|pipe|let|type-alias|decorator)$", "destination": "/api/:package/:api"},
```

Super nit: Order alphabetically:
```suggestion
      {"type": 301, "regex": "^/api/(?P<package>[^/]+)/(?P<api>[^/-]+)-(class|decorator|directive|function|interface|let|pipe|type-alias|var)$", "destination": "/api/:package/:api"},
```